<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ffmpegCmd</class>
 <widget class="QWidget" name="ffmpegCmd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Main">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Top">
     <item>
      <widget class="QGroupBox" name="groupBox_Presets">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>命令记录与预设</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_Presets">
        <item>
         <widget class="QListWidget" name="listWidget_History"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ClearHistory">
          <property name="text">
           <string>清除历史</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_AIAssistant">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>AI 智能助手</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_AI">
        <item>
         <widget class="QLabel" name="label_AIStatus">
          <property name="text">
           <string>AI助手已就绪</string>
          </property>
          <property name="styleSheet">
           <string>color: #4CAF50; font-weight: bold;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textEdit_AIChat">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>AI回复将显示在这里...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_AIInput">
          <property name="placeholderText">
           <string>向AI助手提问...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_AIButtons">
          <item>
           <widget class="QPushButton" name="btn_AISend">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_AIClear">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_AISettings">
            <property name="text">
             <string>设置</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_Command">
       <item>
        <widget class="QGroupBox" name="groupBox_Input">
         <property name="title">
          <string>FFmpeg 命令输入</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_Input">
          <item>
           <widget class="QPlainTextEdit" name="textEdit_Command">
            <property name="placeholderText">
             <string>在此输入 ffmpeg 命令，例如: ffmpeg -i input.mp4 output.avi</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Run">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>执行命令</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Stop">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>停止执行</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Output">
         <property name="title">
          <string>执行输出</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_Output">
          <item>
           <widget class="QPlainTextEdit" name="textEdit_Output">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="backgroundVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
